lib/OpenQA/Worker/Cache.pm

```diff
@@ -35,173 +37,325 @@ our (@ISA, @EXPORT);
 my $cache;
 my $host;
 my $location;
-my $limit = 50;
-my $db_file;
+our $limit = 50 * 1024 * 1024 * 1024;
```

Comment:
Sounds to me like we will run into problems until this is really configurable from the outside